### DISCUSSION TITLE
Is there a repo for plugins?

### DISCUSSION THREAD

**Discussion by EMREOYUN** [Pryaxis/TShock#2148] [Q&A] [answered]
Most of Offical Page plugin does not work and made changes on their source code in order to work. Some of them work but some of them does not. Is there a repo with up-to-date plugins?

**Comment by hakusaro** [comment, marked as answer]
GitHub.com/Pryaxis/Plugins?

**Comment by EMREOYUN** [comment]
Yes, this is what wanted but there was less plugins there. If you know other repos, can you post it there(Same valid for other members of GitHub)?

**Reply by bartico6** [reply]
There aren't any other "centralised" plugin repositories like the old forums as far as I know, except said forums which are deprecated and largely out of date.

**Reply by EMREOYUN** [reply]
> There aren't any other "centralised" plugin repositories like the old forums as far as I know, except said forums which are deprecated and largely out of date.

Alright, thank you very much.
